src/main/resources/assets/merequester/lang/pt_br.json

```diff
@@ -0,0 +1,31 @@
+{
+    "itemGroup.merequester": "ME Solicitante",
+  
+    "item.merequester.requester_terminal": "ME Terminal do Solicitante",
+    "block.merequester.requester": "ME Solicitante",
+  
+    "gui.merequester.requester_terminal": "ME Terminal do Solicitante",

```

Comment:
Have you made sure this fits on the screen? It's the label shown next to the search text box at the top of the terminal.